src/main/haskell/kore/src/Kore/Builtin/Builtin.hs

```diff
@@ -299,6 +276,24 @@ verifySort _ _ (SortVariableSort SortVariable { getSortVariable }) =
     Kore.Error.koreFail
         ("unexpected sort variable '" ++ getIdForError getSortVariable ++ "'")
 
+-- | Wildcard for sort verification on parameterized builtin sorts
+anySort :: MonadVerify m => SortVerifier m
```

Comment:
I would prefer to call this `acceptAnySort`, to make it clear that it is a `SortVerifier` action.